ClntIfaceMgr/ClntIfaceMgr.cpp

```diff
@@ -492,6 +492,12 @@ bool TClntIfaceMgr::modifyPrefix(int iface, SPtr<TIPv6Addr> prefix, int prefixLe
         return false;
     }
 
+    if(strcmp(action.c_str(), "Deleting") == 0) {
```

Comment:
There's several problems with that. First, you can compare the code with `if (action == "Deleting")`. Even easier would be to do `if (mode == PREFIX_MODIFY_DEL)` as enum comparison is simpler, faster and produces smaller binary. There should also be a space between `if` and `(`. But the biggest problem is that the logic is wrong. The log message says the script is going to be invoked since rebinding failed for prefix. This is one of many cases when this code is called. Two others are: when the client is shutting down and release was sent. When the server's policy changed and it assigned a new prefix and the old one was revoked.

Reply:
I have corrected it.

Reply:
I have corrected it.